src/test/java/eu/europa/ted/efx/XPathContextualizerTest.java

```diff
@@ -60,12 +60,12 @@ void testAttributeInXpath() {
 
   @Test
   void testAttributeInContext() {
-    assertEquals("../c/d", contextualize("/a/b/@attribute", "/a/b/c/d"));
+    assertEquals("c/d", contextualize("/a/b/@attribute", "/a/b/c/d"));
```

Comment:
This does not seem correct: an attribute is like a child. So the ../ is needed.
See this example: http://xpather.com/ZjCLbf5X